janus_core/cli/geomopt.py

```diff
@@ -54,6 +57,9 @@ def geomopt(
             help="Fully optimize the cell vectors, angles, and atomic positions.",
         ),
     ] = False,
+    pressure: Annotated[
+        float, Option(help="Scalar pressure when optimizing cell vectors, in bar.")
```

Comment:
this shall apply regardless of what we optimize, full cell or only lengths. also pressure shall be in GPa, mainly cause ase documentation shows that. happy to have it in bar for the right argument.

Reply:
Fully optimizing does optimize vectors though, doesn't it? Whereas this change implies the pressure is always used, which isn't the case if neither flag is chosen.

No objections changing to GPa, I only went for bar since we had it in a few places elsewhere.